services/src/kilda-core/kilda-persistence-neo4j/src/test/java/org/openkilda/persistence/repositories/impl/Neo4jIslRepositoryTest.java

```diff
@@ -335,6 +338,10 @@ public void shouldDeleteIsl() {
         islRepository.delete(isl);
 
         assertEquals(0, islRepository.findAll().size());
+        assertEquals(2, switchRepository.findAll().size());
+
+        switchRepository.delete(switchA);
```

Comment:
No need to delete after each test if there's shared Neo4jBasedTest.cleanUpTestServer().

Reply:
Applied.